packages/components/src/DateTimePickers/Time/Input/Input.scss

```diff
@@ -0,0 +1,14 @@
+.time-picker-input {
+	display: flex;
+	align-items: flex-end;
+
+	:global(.form-control) {
+		margin-right: $padding-small;
+	}
+
+	svg {
+		width: 1.4rem;
+		height: 1.4rem;
+		margin-bottom: 0.2rem;
```

Comment:
The icon is not well aligned for every font size (user can change the font size with accessibility features.
What you need to do: 
- .time-picker-input : align items to `center`
- svg : remove the margin
- on the tooltip div wrapper: display block